server/channels/app/integration_action.go

```diff
@@ -40,12 +41,7 @@ import (
 	"github.com/mattermost/mattermost/server/v8/channels/utils"
 )
 
-func (a *App) DoPostAction(c *request.Context, postID, actionId, userID, selectedOption string) (string, *model.AppError) {
```

Comment:
q: is it intentional the removal of this function? if so, maybe the documentation in the top of this file should be updated then

Reply:
e43bca73a5

Reply:
Yes, the removal is intentional. `DoPostAction` is not longer called in the codebase.